api/src/pcapi/routes/backoffice/offerers/validation_blueprint.py

```diff
@@ -235,7 +234,7 @@ def _offerer_batch_action(
 
     if hasattr(form, "tags"):
         tags = form.tags.data
-        previous_tags = list(reduce(set.intersection, [set(offerer.tags) for offerer in offerers]))  # type: ignore [arg-type]
+        previous_tags = list(set.intersection(*[set(offerer.tags) for offerer in offerers]))
         deleted_tags = list(set(previous_tags).difference(list(set(tags))))
```

Comment:
Au passage, je me demande à quoi sert l'appel à `list` en fin d'expression. On devrait pouvoir écrire : 

```suggestion
        deleted_tags = list(set(previous_tags).difference(set(tags)))
```

Non ?